ext/json/json_parser.y

```diff
@@ -307,7 +349,8 @@ PHP_JSON_API void php_json_parser_init(php_json_parser *parser,
 		const char *str,
 		size_t str_len,
 		int options,
-		int max_depth)
+		int max_depth,
+		bool validate_only)
```

Comment:
Could/should `validate_only` not be rolled into the `options` field?

Reply:
I will check this and will be back to you

Reply:
@derickr 

I was able to inject a constant value into options, and ask for it later in the implementation.

File: `php_json.h`
![image](https://user-images.githubusercontent.com/25756860/186898872-45dc97d1-b5e0-4af7-bec8-7dc2aa20a8db.png)


The rest of the implementation just asks for it, should be injected before calling `php_json_parser_init` like in:
![image](https://user-images.githubusercontent.com/25756860/186899121-0d5e5baa-fb90-4eff-9a50-fdea3338753b.png)


Is it ok? (I liked it actually)

Reply:
I did it like this.